ups/ap_pipe.table

```diff
@@ -10,7 +10,6 @@ setupRequired(utils >= 4.6.0.0)
 setupRequired(pex_exceptions >= 4.6.0.0)
 setupRequired(ndarray)
 
-setupRequired(obs_decam)
```

Comment:
I don't think you depend on ndarray, pex_exceptions, or boost either so please remove them.